src/Tools/Test Framework/Test Runner/src/TestSuiteMgt.Codeunit.al

```diff
@@ -309,6 +310,33 @@ codeunit 130456 "Test Suite Mgt."
         GetTestMethods(ALTestSuite, CodeunitMetadata);
     end;
 
+    internal procedure SelectTestMethodsByExtensionAndTestType(var ALTestSuite: Record "AL Test Suite"; ExtensionID: Text; TestType: Text)
```

Comment:
Why is this not a guid? Then you don't need to evaluate it to a guid either. You can still check if it's a null guid